lms/static/sass/discussion/views/_inline.scss

```diff
@@ -0,0 +1,63 @@
+// forums - inline discussion styling
+// ====================
+
+.discussion.inline-discussion {
+    padding-top: $baseline;
+
+    .inline-threads {
+        border: 1px solid $forum-color-border;
+        border-radius: $forum-border-radius;
+    }
+
+    .inline-thread {
+        border: 1px solid $forum-color-border;
+        border-radius: $forum-border-radius;
+
+        .forum-nav-bar {
+            background-color: $forum-color-navigation-bar;
```

Comment:
I think we should minimize use of blue - we want it only to call attention to actionable options.
My recommendation would be to make the link blue (Sass: palette(primary, base) RGB: rgb(0, 117, 180) HEX: 0075b4) and background x-back grey (Sass: palette(grayscale, x-back) RGB: rgb(245, 245, 245) HEX: f5f5f5).

I agree with @marcotuts padding recommendations as well.